SupportClient/SupportClient/Views/FileCollectionViewCell.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ErrorFileCollectionViewCell.swift
+//  SupportClient
+//
+//  Created by Cameron Russell on 3/7/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import Photos
+import UIKit
+
+protocol RemoveFileDelegate: class {
+    func removeFile(_ file: PHAsset)
+}
+
+class FileCollectionViewCell: UICollectionViewCell, UIImagePickerControllerDelegate & UINavigationControllerDelegate {
+    
+    static let reuseID = "ErrorFileCollectionViewCell"
+    
+    weak var delegate: RemoveFileDelegate?
+
+    private let imageManager = PHCachingImageManager()
+    private let fileImageView = UIImageView()
+    private let removeFileButton = MaskedButton()
+    
+    private var rawPHAsset = PHAsset()
+    private let padding: CGFloat = 20
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupImageView()
+        setupRemoveFileButton()
+        setupContraints()
+    }
+    
+    func setupImageView() {
+        fileImageView.translatesAutoresizingMaskIntoConstraints = false
+        fileImageView.contentMode = .scaleAspectFit
+        fileImageView.layer.borderWidth = 1.0
+        fileImageView.layer.borderColor = UIColor.black.cgColor
+        fileImageView.layer.cornerRadius = 8
+        fileImageView.layer.masksToBounds = true
+        fileImageView.isUserInteractionEnabled = true
+        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(imageTapped(tapGestureRecognizer:)))
+        fileImageView.isUserInteractionEnabled = true
+        fileImageView.addGestureRecognizer(tapGestureRecognizer)
+        contentView.addSubview(fileImageView)
+    }
+    
+    func setupRemoveFileButton() {
+        removeFileButton.translatesAutoresizingMaskIntoConstraints = false
+        removeFileButton.setTitle("✕", for: .normal)
+        removeFileButton.backgroundColor = .red
+        removeFileButton.contentHorizontalAlignment = .center
+        removeFileButton.contentVerticalAlignment = .center
```

Comment:
If i understand correctly, `.center` should be the default already so no need to explicitly set here